src/exercises/day-3/safe-ffi-wrapper.md

```diff
@@ -2,7 +2,7 @@
 
 Rust has great support for calling functions through a _foreign function
 interface_ (FFI). We will use this to build a safe wrapper for the `libc`
-functions you would use from C to read the filenames of a directory.
+functions you would use from C to read the names of files in a directory.
```

Comment:
Please take out this change. I think it has already been merged by now in another PR :smile:

Reply:
I don't know how to do that.

Reply:
You probably just need to merge `main` into your branch.

In the future, you should start new branches from the latest version on `main`. Lots of it is automatic if you use the little pencil icon in the top-right of any page to edit the page:

![image](https://github.com/google/comprehensive-rust/assets/89623/913a0f01-ee45-4aea-8b9e-ce85103bb292)